qa/acceptance-tests/src/test/java/io/camunda/it/client/UsageMetricsTest.java

```diff
@@ -80,25 +84,38 @@ static void setup(@Authenticated(ADMIN) final CamundaClient adminClient)
       throws InterruptedException {
     createTenant(adminClient, TENANT_A);
     createTenant(adminClient, TENANT_B);
+    createTenant(adminClient, TENANT_C);
     assignUserToTenant(adminClient, ADMIN, TENANT_A);
     assignUserToTenant(adminClient, ADMIN, TENANT_B);
+    assignUserToTenant(adminClient, ADMIN, TENANT_C);
 
     // Create PI & wait for metrics to be exported
     deployResource(adminClient, "process/service_tasks_v1.bpmn", TENANT_A);
+    deployResource(adminClient, "process/process_with_assigned_user_task.bpmn", TENANT_A);
     startProcessInstance(adminClient, PROCESS_ID, TENANT_A);
+    startProcessInstance(adminClient, PROCESS_ID_2, TENANT_A);
     waitForUsageMetrics(
         adminClient,
         NOW.minusDays(1),
         NOW.plusDays(1),
-        res -> assertThat(res.getProcessInstances()).isEqualTo(1));
+        res -> {
+          assertThat(res.getProcessInstances()).isEqualTo(2);
+          assertThat(res.getAssignees()).isEqualTo(1);
+          assertThat(res.getActiveTenants()).isEqualTo(1);
+        });
 
     // Store first export time & wait 2 export intervals
     exportedTime = OffsetDateTime.now();
     Thread.sleep(2 * EXPORT_INTERVAL.toMillis());
 
     // Create PI for TENANT_B
     deployResource(adminClient, "process/service_tasks_v1.bpmn", TENANT_B);
+    deployResource(adminClient, "process/process_with_assigned_user_task.bpmn", TENANT_B);
+    deployResource(adminClient, "process/process_with_assigned_user_task.bpmn", TENANT_C);
     startProcessInstance(adminClient, PROCESS_ID, TENANT_B);
+    startProcessInstance(adminClient, PROCESS_ID_2, TENANT_B);
```

Comment:
🔧 Could we start it twice on the same tenant? Just to show that the `withThenants` search also filters out duplicates.